tests/test_cli_compile.py

```diff
@@ -1393,16 +1393,6 @@ def test_stdin_without_output_file(runner):
     assert "--output-file is required if input is from stdin" in out.stderr
 
 
-def test_not_specified_input_file(runner):
```

Comment:
I'd keep this as a separate test but rename it to specify that it tests the case when both the input isn't passed and the none of the fallback files are present on disk.
It is separate from testing that the input isn't passed but one (or more) of the fallback files exists.